staging/src/k8s.io/apiserver/pkg/endpoints/handlers/get.go

```diff
@@ -107,7 +107,7 @@ func GetResource(r rest.Getter, scope *RequestScope) http.HandlerFunc {
 					return nil, err
 				}
 			}
-			tracing.SpanFromContext(ctx).AddEvent("About to Get from storage")
+			tracing.SpanFromContext(ctx).AddEvent("About to Get from cacher")
```

Comment:
I don't agree with this change - at this point we don't know if it will be server from cacher or from etcd.

Please revert here and below.

Reply:
It would first hit cacher at https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/apiserver/pkg/storage/cacher/cacher.go#L695 and only afterwards we'll get the event if it hits actual storage, am I understanding this correctly?

Reply:
if cacher is enabled (e.g. it's not enabled for events by default)

also - cacher is an implementation of storage, so let's not change that.

Reply:
oh, so "storage" in this context is "storage implementation, which starts with cacher" and not "underlying storage (read: etcd)"? Its a bit confusing that the next span would be "cacher.GetList" though. 

How about:
* `About to Get from cacher` here
* cacher.Get or cacher.List is called
* if cache is uninitialized or missed the event `About to Get from storage implementation` is emitted?

Reply:
Cacher doesn't have to be there - and by default it's not enabled for events.
We shouldn't mention cacher here.

Reply:
Updated, PTAL